Dockerfile.testing

```diff
@@ -0,0 +1,82 @@
+FROM golang:1.12-stretch
+MAINTAINER TRON-US <support@tron.network>
```

Comment:
Do we manage that email address? It would be useful to track it somehow.

Reply:
i am not sure.  it was the email in the original Dockerfile and i left it alone.  :)